workspaces/global-header/plugins/global-header/app-config.dynamic.yaml

```
@@ -7,4 +7,47 @@ dynamicPlugins:
           importName: GlobalHeader
           config:
             layout:
-              position: above-main-content
+              position: above-main-content # above-main-content | below-main-content
+        - mountPoint: application/header/component
+          importName: SearchComponent
```

I thought that we have a mount point that make it clear:

1. application/header => rhdh
2. global.header/component => global-header-plugin

So I would change it to:

```suggestion
        - mountPoint: global.header/component
          importName: SearchComponent
```